### DISCUSSION TITLE
How does `robyn` compare to `FastAPI`?

### DISCUSSION THREAD

**Discussion by asaah18** [sparckles/Robyn#147] [Q&A] [answered]
Does both have the same object? Or they aiming to achieve different things?

**Comment by sansyrox** [comment, marked as answer]
Hi @asaah18 ,

Thank you for your question. 

While both FastAPI and Robyn are aiming to improve the Python web ecosystem, there are a few differences:

- Robyn's response time is much faster than that of FastAPI.
- FastAPI relies on ASGIs while you do not require any SGI for Robyn.
- Robyn has a rust runtime whereas FastAPI has a Python only ecosystem.
- Robyn is still a very young project while FastAPI has a comparatively mature ecosystem.

These are some of the differences from the top of my head.
